src/containers/App/App.js

```diff
@@ -1,29 +1,41 @@
 import React from 'react';
 import { func } from 'prop-types';
 import { translate } from 'react-i18next';
+import { Router } from '@reach/router';
+import { Home } from '../../pages';
 import 'i18n/i18n';
 import styles from './assets/app.scss';
 
-export function App({ t }) {
+// export function App({ t }) {
+//   return (
+//     <div className={styles.app}>
+//       <img src={require('./assets/mentor.jpg')} alt="Mentor" className={styles.image} />
+//       <header className={styles.header}>
+//         <h1 className={styles.title}>{t('landing-page-header')}</h1>
+//         <h4 className={styles.tagline}>{t('landing-page-subheader')}</h4>
+//         {/* TODO: create a React Component for a `Button` and add scoped styles */}
+//         <a
+//           className="button button__white--fill button__link"
+//           href="https://docs.google.com/forms/d/1TraoygcJay39cLiBQM-ROlToy7xIgcdx_VvCQuE1lJw/edit#responses"
+//           target="_blank"
+//           rel="noopener noreferrer"
+//         >
+//           {t('landing-page-start')}
+//         </a>
+//       </header>
+//     </div>
+//   );
+// }
+
+const App = ({ t }) => {
   return (
-    <div className={styles.app}>
-      <img src={require('./assets/mentor.jpg')} alt="Mentor" className={styles.image} />
-      <header className={styles.header}>
-        <h1 className={styles.title}>{t('landing-page-header')}</h1>
-        <h4 className={styles.tagline}>{t('landing-page-subheader')}</h4>
-        {/* TODO: create a React Component for a `Button` and add scoped styles */}
-        <a
-          className="button button__white--fill button__link"
-          href="https://docs.google.com/forms/d/1TraoygcJay39cLiBQM-ROlToy7xIgcdx_VvCQuE1lJw/edit#responses"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          {t('landing-page-start')}
-        </a>
-      </header>
-    </div>
+    <React.Fragment>
```

Comment:
Do we really need the `React.Fragment` here?

Reply:
It's better than a placeholder div, no? That was why the React team introduced it. https://reactjs.org/docs/fragments.html

Reply:
Yes it is! But I'm just wondering if <Router> is enough. I haven't used reach router, that's what I'm asking.

Reply:
Oh crap. It is. Sorry about this!